var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -1048,6 +1055,7 @@ def cmake_args(self):
             self.define_from_variant("CP2K_ENABLE_GRID_GPU", "grid_gpu"),
             self.define_from_variant("CP2K_ENABLE_DBM_GPU", "dbm_gpu"),
             self.define_from_variant("CP2K_ENABLE_PW_GPU", "pw_gpu"),
+            self.defile_from_variant("CP2K_USE_GRPP", "grpp"),
```

Comment:
```suggestion
            self.define_from_variant("CP2K_USE_GRPP", "grpp"),
```
It would be useful to always test locally, to make sure everything works. =)

Reply:
sorry, I corrected the typo